src/workflow/ArcanistDiffWorkflow.php

```diff
@@ -760,7 +761,7 @@ private function displayGitHubInvitationBanner() {
     
     // Only show the prompt for new revisions (not when updating existing ones)
     // We need to check multiple conditions to determine the actual intent
-    if ($this->shouldPromptForArh()) {
+    if ($this->shouldPromptForArh() && $this->isRepositoryAllowedForGitHubPrompt()) {
```

Comment:
Can we do this same check for the banner?  I'm concerned we're going to get a lot of noise around GH + micro-repos.